packages/apollo-server-lambda/src/lambdaApollo.ts

```diff
@@ -38,13 +38,20 @@ export function graphqlLambda(
         statusCode: 500,
       });
     }
+
+    let query: any = event.queryStringParameters;
+    if (event.httpMethod === 'POST' && event.body) {
+      if (typeof event.body === 'string') {
+        query = JSON.parse(event.body);
```

Comment:
This will thow an error when I try use apollo-client-upload. Since the content-type is multipart, event.body is a string but not a json.  Not sure how can this be solved ? Or will this support multipart ?